src/encoded/static/components/item-pages/ExperimentView.js

```diff
@@ -67,14 +73,20 @@ export default class ExperimentView extends WorkflowRunTracingView {
             });
         }
 
-        const procesedFilesWithViewPermissions = ExperimentView.procesedFilesWithViewPermissions(context);
+        const processedFilesWithViewPermissions = ExperimentView.processedFilesWithViewPermissions(context);
+        const processedFilesLen = (processedFilesWithViewPermissions && processedFilesWithViewPermissions.length) || 0;
 
-        if (procesedFilesWithViewPermissions && procesedFilesWithViewPermissions.length > 0) {
+        if (processedFilesLen > 0) {
             tabs.push({
-                tab : <span><i className="icon icon-microchip fas icon-fw"/> Processed Files</span>,
+                tab : (
+                    <span>
+                        <i className="icon icon-microchip fas icon-fw"/>
+                        { processedFilesLen + " Processed File" + (processedFilesLen === 1 ? "" : "s") }
+                    </span>
+                ),
```

Comment:
We now show "N Raw Files" instead of "Raw Files" as tab title. Because we can and its more upfront info.